cmd/test-utils/common.go

```diff
@@ -18,31 +18,27 @@ import (
 
 // TODO (carlosms) this could be build/bin, workaround for https://github.com/src-d/ci/issues/97
 var srcdBin = fmt.Sprintf("../../../build/engine_%s_%s/srcd", runtime.GOOS, runtime.GOARCH)
+var srcdPrevBin = fmt.Sprintf("../../../build-prev/engine_%s_%s/srcd", runtime.GOOS, runtime.GOARCH)
+var srcdNextBin = fmt.Sprintf("../../../build-next/engine_%s_%s/srcd", runtime.GOOS, runtime.GOARCH)
 var configFile = "../../../integration-testing-config.yaml"
 
-type IntegrationSuite struct {
-	suite.Suite
-}
-
 func init() {
 	if os.Getenv("SRCD_BIN") != "" {
 		srcdBin = os.Getenv("SRCD_BIN")
 	}
-}
 
-func (s *IntegrationSuite) CommandContext(ctx context.Context, cmd string, args ...string) *exec.Cmd {
-	args = append([]string{cmd}, args...)
-	return exec.CommandContext(ctx, srcdBin, args...)
+	if os.Getenv("SRCD_PREV_BIN") != "" {
+		srcdPrevBin = os.Getenv("SRCD_PREV_BIN")
+	}
 }
 
-func (s *IntegrationSuite) RunCommand(ctx context.Context, cmd string, args ...string) (*bytes.Buffer, error) {
-	var out bytes.Buffer
-
-	command := s.CommandContext(ctx, cmd, args...)
-	command.Stdout = &out
-	command.Stderr = &out
+type IntegrationSuite struct {
+	suite.Suite
+	Cmd *Commander
```

Comment:
I think this could have been embedded to avoid many changes to the existing tests
```go
type IntegrationSuite struct {
	suite.Suite
	Commander
}
```

Reply:
Woa! For some reason I was thinking that only one struct can be embedded! 😂
Yeah that should minimize the changes!